src/astro/constants.py

```diff
@@ -36,3 +36,13 @@ class Database(Enum):
 SUPPORTED_DATABASES = list({const.value for const in Database})
 
 UNIQUE_TABLE_NAME_LENGTH = 63
+
```

Comment:
can we use aenum package(https://pypi.org/project/aenum/). It will simplify the code.

Reply:
TBH I'm not sure about the stability of the project.

Reply:
Naah, don't think there is enough value to add an external dependency